storage/rocksdb/rdb_datadic.h

```diff
@@ -113,6 +113,9 @@ const size_t RDB_UNPACK_DATA_LEN_SIZE = sizeof(uint16_t);
 const size_t RDB_UNPACK_HEADER_SIZE =
     sizeof(RDB_UNPACK_DATA_TAG) + RDB_UNPACK_DATA_LEN_SIZE;
 
+const char RDB_UNPACK_COVERED_DATA_TAG = 0x03;
```

Comment:
Can you add a comment about the format here? It's a 1 tag-byte plus a two byte length field plus a two byte covered bitmap. Can you also define a constant for RDB_UNPACK_COVERED_HEADER_SIZE that is sizeof(RDB_UNPACK_COVERED_DATA_TAG) + RDB_UNPACK_COVERED_DATA_LEN_SIZE + RDB_COVERED_BITMAP_SIZE and use this constant where you are reading/writing RDB_UNPACK_COVERED_DATA_TAG? Even though there are similarities in the format, it makes the two code paths explicit for header construction and one does not depend on the other.